src/main/resources/META-INF/plugin.xml

```diff
@@ -169,5 +169,9 @@ limitations under the License.
             <add-to-group
                     group-id="EditorPopupMenu" anchor="first" />
         </action>
+
+        <action id="se.isselab.HAnS.TestDepthMetrics" class="se.isselab.HAnS.TestDepthMetrics" text="testingMetrics">
```

Comment:
is this to trigger the test? if so should not be in the plugin.xml.

Reply:
yes, where should i move it?